include/cuco/static_multimap.cuh

```diff
@@ -23,6 +23,7 @@
 #include <cuco/probe_sequences.cuh>
 #include <cuco/traits.hpp>
 
+#include <thrust/for_each.h>
```

Comment:
```suggestion
```
Unnecessary since it's only appeared in docs: https://github.com/NVIDIA/cuCollections/blob/4d36c90621d5a4165a7f61788a53b77db767f431/include/cuco/static_multimap.cuh#L649-L657

Reply:
I tried not to insert includes for algorithms mentioned in docstrings but it was a manual process to verify each one, so I am not surprised that some slipped through. 😬